interfaces/builtin/opengl.go

```diff
@@ -83,6 +83,9 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 /dev/nvhost-* rw,
 /dev/nvmap rw,
 
+# Tegra display driver
+/dev/tegra_dc_* rw,
```

Comment:
What does this match in practice?

Reply:
`tegra_dc_ctrl`, `tegra_dc_0` and `tegra_dc_1` (control device plus additional devices per display you can connect).

Reply:
I have created slightly more fine-grained rules.